tests/Consumer/AbstractMailjetConsumerTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+namespace Tests\AppBundle\Consumer;
+
+use AppBundle\Repository\MailjetEmailRepository;
+use Doctrine\Bundle\DoctrineBundle\Registry;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Persistence\ObjectManager;
+use OldSound\RabbitMqBundle\RabbitMq\ConsumerInterface;
+use PhpAmqpLib\Message\AMQPMessage;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+class AbstractMailjetConsumerTest extends TestCase
+{
+    const CLASS_NAME = 'AppBundle\Consumer\AbstractMailjetConsumer';
```

Comment:
It's better to use an abstract protected method that relying on a constant definition.